lineapy/execution/executor.py

```diff
@@ -239,6 +241,36 @@ def _execute(
             f"Don't know how to execute node type {type(node)}"
         )
 
+    @_execute.register
+    def _execute_if(
+        self,
+        node: IfNode,
+        changes: Iterable[TracebackChange],
+        variables: Optional[Dict[str, LineaID]],
+    ) -> PrivateExecuteResult:
+        return PrivateExecuteResult(
+            # An `execute if` uses the test condition to create a new control flow node  a global set by a call node so,
+            # Copy the result and the timing from the call node
+            self._id_to_value[node.test_id],
+            *self._execution_time[node.test_id],
+            [],
+        )
+
+    @_execute.register
+    def _execute_else(
+        self,
+        node: ElseNode,
+        changes: Iterable[TracebackChange],
+        variables: Optional[Dict[str, LineaID]],
+    ) -> PrivateExecuteResult:
+        return PrivateExecuteResult(
+            # An `execute else` simply creates a new control flow node using no other variables,
+            # Copy the result and the timing from the attached `if` or `for` or `while` node
+            self._execution_time[node.companion_id],
```

Comment:
should not return execution time in the place of value. the value of else node is None so should say that here.